packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1297,7 +1297,14 @@ const messages = {
         term2: 'Ikke skriv noe støtende i tekstfelt.',
         term3: 'NDLA forbeholder seg retten til å oppdatere eller slette utdaterte ressurser.',
       },
-      newFavourite: 'Nylig lagt til',
+      recentFavourites: {
+        title: 'Nylig til i mine favoritter',
```

Comment:
Ser ut som om du har glømt et ord et par plasser.
```suggestion
        title: 'Nylig lagt til i mine favoritter',
```